apps/financial-aid/web-veita/src/utils/useApplicationState.ts

```diff
@@ -45,14 +42,10 @@ export const useApplicationState = () => {
       })
 
       if (data) {
-        if (applicationFilters && setApplicationFilters) {
-          setApplicationFilters((preState) => ({
-            ...preState,
-            [prevState]: applicationFilters[prevState] - 1,
-            [state]: applicationFilters[state] + 1,
-          }))
+        if (data.updateApplication.filters) {
+          setApplicationFilters(data.updateApplication.filters)
+          delete data.updateApplication.filters
```

Comment:
Why do we delete them? Is this because of some endless update or?

Reply:
I delete it because I already used it and I dont want to inject it into the application, maybe it is unnecessary?